rskj-core/src/main/java/co/rsk/peg/BridgeStorageIndexKey.java

```diff
@@ -24,7 +24,7 @@ public enum BridgeStorageIndexKey {
     PEGOUT_TX_SIG_HASH("pegoutTxSigHash"),
 
     SVP_FUND_TX_HASH_UNSIGNED("svpFundTxHashUnsigned"),
-    SVP_FUND_TX_HASH_SIGNED("svpFundTxHashSigned"),
+    SVP_FUND_TX_SIGNED("svpFundTxSigned"),
```

Comment:
Do we need to update the RSKIP?

Reply:
yes, indeed! I'll create a task for it, thanks!!!